projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.spec.ts

```diff
@@ -91,6 +91,7 @@ describe('ProductFacetNavigationComponent in product-list', () => {
           provide: ActivatedRoute,
           useClass: MockActivatedRoute,
         },
+        { provide: ModalService, useValue: { open: () => {} } },
```

Comment:
To be more consistent with other unit tests my suggestion is to create a mock for ModalService class, instead of using custom object here.

Reply:
Since we currently have only one method, I would leave this as is. additionally, before replacing ngbModalService with our own service, ngb was actually done this way (with arrow function). so let's focus on changing unit tests when we remove ngbModal in the future :)